Charith/example-5/data-gathering.html

```diff
@@ -11,51 +12,112 @@
           integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 </head>
 <body>
-<div class="bg-info">
     <div class="bg-info">
-        <h2 class="text-uppercase text-center text-white pb-3">Simple Contact Form</h2>
-    </div>
-    <div class="card bg-light w-50 mx-auto">
-        <div class="card-body">
-            <form class="bg-light p-3">
-                <div class="form-group text-danger"><label for="first-name">FIRST NAME</label>
-                    <input type="text" class="form-control" id="first-name" placeholder="FIRST NAME" name="first name"
-                           required>
-                </div>
-                <div class="form-group text-danger"><label for="LAST NAME">LAST NAME</label>
-                    <input type="text" class="form-control" id="LAST NAME" placeholder="LAST NAME" name="LAST NAME"
-                           required>
-                </div>
-                <div class="form-group text-danger"><label for="EMAIL">EMAIL</label>
-                    <input type="text" class="form-control" id="EMAIL" placeholder="EMAIL" name="EMAIL" required>
-                </div>
-                <div class="form-group text-danger"><label for="PHONE NUMBER">PHONE NUMBER</label>
-                    <input type="text" class="form-control" id="PHONE NUMBER" placeholder="PHONE NUMBER"
-                           name="PHONE NUMBER"
-                           required>
-                </div>
-                <div class="form-group text-danger"><label for="CITY">CITY</label>
-                    <input type="text" class="form-control" id="CITY" placeholder="CITY" name="CITY" required>
-                </div>
-                <div class="form-group text-danger">
-                    <label for="MESSAGE">MESSAGE</label>
-                    <textarea class="form-control" id="MESSAGE" rows="3" placeholder="MESSGAE"></textarea>
-                </div>
-                <button class="text-center bg-info w-100 text-white">Send</button>
-            </form>
+
+        <!--header text area begin-->
+        <div class="bg-info">
+            <h2 class="text-uppercase text-center text-white pb-3">Simple Contact Form</h2>
+        </div>
+        <!--header text area end-->
+
+        <!--    form area begin-->
+        <div class="card bg-light w-50 mx-auto">
+            <div class="card-body">
+                <form class="bg-light p-3" action="" name = "myForm" onsubmit = "return(validate());">
+                    <div class="form-group text-danger"><label for="first-name">FIRST NAME</label>
```

Comment:
Not sure why did you use 'text-danger' class for the label. If the label is red on a given design that is fine. But use it only for the label element.